api/pom.xml

```diff
@@ -193,9 +193,27 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>2.8</version>
+                <version>3.1.0</version>
                 <configuration>
+                    <source>8</source>
                     <additionalparam>-Xdoclint:none</additionalparam>
+                    <doctitle>Jakarta Deployment ${project.version} API</doctitle>
+                    <sourceFileExcludes>
+                        <sourceFileExclude>**/module-info.java</sourceFileExclude>
+                        <sourceFileExclude>target/**/*.java</sourceFileExclude>
+                    </sourceFileExcludes>
+                    <docfilessubdirs>true</docfilessubdirs>
+                    <links>
+                        <link>http://docs.oracle.com/javase/8/docs/api/</link>
+                    </links>
+                    <detectJavaApiLink>false</detectJavaApiLink>
+                    <detectOfflineLinks>false</detectOfflineLinks>
+                    <header><![CDATA[<br>Jakarta Deployment API v${project.version}]]>
+                    </header>
+                    <bottom><![CDATA[
+                    Copyright &#169; {inceptionYear}&#x2013;{currentYear} Oracle and/or its affiliates. All rights reserved.<br>
```

Comment:
Where are {inceptionYear} and {currentYear} set, and how do they get expanded here?

What is "&#x2013;"?

Reply:
`inceptionYear` is set in EE4J parent.pom. The default value is 2017 which is wrong for this API. I fixed it ny overriding the value in `pom.xml`. `currentYear` is provided by Maven. More info [here](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#bottom).

Reply:
My previous comment asking about the html escape didn't come through properly.
It looks like you're using an html escape for a long dash between years, but the
Eclipse recommended separator is a comma.

Where's the spec for these things that look like property references but don't start
with dollar sign?  Can I use any property?  Why no dollar sign?

Reply:
You can use ${property}, but you will need to have `<property>xxx</property>` defined in `<properties>` section of your `pom.xml`. Without a dollar sign it takes `<inceptionYear>` from the root of your `pom.xml` which is the right place.